.github/workflows/conformance-k8s-kind.yaml

```diff
@@ -147,14 +148,8 @@ jobs:
             --helm-set=cni.chainingMode=portmap \
             --helm-set-string=kubeProxyReplacement=strict \
             --helm-set=sessionAffinity=true \
-            --rollback=false \
-            --config monitor-aggregation=none \
-            --version="
-          HUBBLE_ENABLE_DEFAULTS="--chart-directory=install/kubernetes/cilium \
-            --relay-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci:${SHA} \
-            --relay-version=${SHA}"
+            --helm-set=bpf.monitorAggregation=none"
```

Comment:
you also need to set hubble relay enabled flag as well.

Reply:
this workflow doesn't use hubble relay right? 🤔 https://github.com/cilium/cilium/blob/main/.github/workflows/conformance-k8s-kind.yaml

Reply:
you are right, seems like the hubble_enable_defaults env is not used :face_exhaling: